### DISCUSSION TITLE
my wallets is syncing,but full node Synced.why?how to solved it

### DISCUSSION THREAD

**Discussion by tomuGo** [Chia-Network/chia-blockchain#6408] [Q&A]
![image](https://user-images.githubusercontent.com/24789356/120743104-e79c2380-c52a-11eb-82e9-ac83f5751c12.png)
![image](https://user-images.githubusercontent.com/24789356/120743118-f1be2200-c52a-11eb-8ebe-2b7ed1fa9b0b.png)
![image](https://user-images.githubusercontent.com/24789356/120743143-fedb1100-c52a-11eb-9775-4b2a7691a75d.png)

**Comment by XIII39** [comment]
What version of the Chia software do you have installed?
I found that 1.1.5 stalled like this often. If you haven't already, upgrade to 1.1.6

**Reply by tomuGo** [reply]
> What version of the Chia software do you have installed?
> I found that 1.1.5 stalled like this often. If you haven't already, upgrade to 1.1.6

i use version1.1.6

**Comment by cccat6** [comment]
It is ok. Full node and Wallet are two different things and not sharing the database. Your wallet will sync from the full node you connected with. Usually is your local machine.
Just wait for wallet syncing.

**Reply by tomuGo** [reply]
> machine

but 1 day passed,the height still 251788

**Reply by cccat6** [reply]
Yes, wallet syncing is slow, and you have to wait. Since the wallet is different from different keys, you can not copy the db file as the full node can do.
Syncing wallet will use a full core CPU time. So if you see there is a core usually full, that is wallet syncing.
Just be patient. More than 1 day of waiting is normal.

**Reply by tomuGo** [reply]
> Yes, wallet syncing is slow, and you have to wait. Since the wallet is different from different keys, you can not copy the db file as the full node can do.
> Syncing wallet will use a full core CPU time. So if you see there is a core usually full, that is wallet syncing.
> Just be patient. More than 1 day of waiting is normal.

ok。think u
